plugins/woocommerce/client/admin/client/wp-admin-scripts/command-palette/use-edited-post-type.js

```diff
@@ -4,32 +4,11 @@
 import { useSelect } from '@wordpress/data';
 
 export const useEditedPostType = () => {
-	// This will return 'post' or 'page' when in the post/page editor.
-	const { currentPostType } = useSelect( ( select ) => {
-		const store = select( 'core/editor' );
-		if ( ! store ) {
-			return {
-				currentPostType: null,
-			};
-		}
-		const { getCurrentPostType } = store;
-		return {
-			currentPostType: getCurrentPostType(),
-		};
-	} );
-	// This will return 'wp_template' or 'wp_template_part' when in the Site Editor.
 	const { editedPostType } = useSelect( ( select ) => {
-		const store = select( 'core/edit-site' );
-		if ( ! store ) {
-			return {
-				editedPostType: null,
-			};
-		}
-		const { getEditedPostType } = store;
 		return {
-			editedPostType: getEditedPostType(),
+			editedPostType: select( 'core/editor' ).getCurrentPostType(),
```

Comment:
It returns: `post`, `page`, `wp_template` or `wp_template_part` as the previous two stores would.

Reply:
NIT: can we import the store name from the `@wordpress/editor` package?


```suggestion
			editedPostType: select( editorStore ).getCurrentPostType(),
```

Reply:
Not sure why I didn't in a first place 🙄 Done in https://github.com/woocommerce/woocommerce/pull/60541/commits/5c0d703f9f158bd15734549a96b02e3ddfb64034